pgmpy/models/MarkovChain.py

```diff
@@ -177,7 +177,7 @@ def add_transition_model(self, variable, transition_model):
         -----------
         variable: any hashable python object
             must be an existing variable of the model.
-        transition_model: dict
+        transition_model: dict or ndarray
```

Comment:
@kris-singh I think this line should mention about 2-D array. And the description should mention the structure of the dict or array that we are expecting. Check other parts of the code for examples.